_apps/remove-outside-collaborators.md

```diff
@@ -0,0 +1,78 @@
+---
+# A human-friendly name of your listing
```

Comment:
Could you please remove these comments from the frontmatter?

Reply:
Ooops I could've sworn I did. Yup!